package.json

```diff
@@ -5,16 +5,16 @@
   "scripts": {
     "format": "prettier --write ./",
     "format:check": "prettier --check ./",
-    "spell-check": "cspell  './**'",
+    "spell-check": "cspell  \"./**\"",
     "lint:ls": "ls-lint",
     "lint:md": "markdownlint --ignore node_modules --config ./.markdownlint.jsonc --fix ./ "
   },
   "author": "HackYourFuture Belgium",
```

Comment:
Change to DeNepo since the license has changed?

Reply:
Done.